fetch.bs

```diff
@@ -2213,6 +2346,88 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<p>The requirements for <dfn>recording connection timing info</dfn> given a <var>connection</var>
+and its <a for=/>connection timing info</a> <var>timingInfo</var>, are as follows:
```

Comment:
I find this a bit confusing as it's not an argument, but I don't really have a good suggestion for how to address it either. 😟

Reply:
The requirements are actually written in the order they should happen in the connection sequence, so I think changing it to an `ol` with a `let` in the beginning wouldn't be so bad.

Reply:
Let's leave this until the more structural issues are sorted.